warmenu.lua

```diff
@@ -53,6 +53,12 @@ local defaultStyle = {
 	buttonPressedSound = { name = 'SELECT', set = 'HUD_FRONTEND_DEFAULT_SOUNDSET' }, --https://pastebin.com/0neZdsZ5
 }
 
+local lastScroll = {}
```

Comment:
`lastScrollTimers`?

Reply:
Changed to `lastScrollTimes` in 508d768 since I like that best describes the contents of the table